### ISSUE TITLE
Reducing height of tplot window eventually makes all y-axis numerical values disappear.

### ISSUE THREAD

**Issue by phansel** [MAVENSDC/PyTplot#131]
This makes the utility unusable for more than ~5 vertically stacked charts. Cannot seem to locate the option to increase numerical label density.

**Comment by bryan-harter** [comment]
I just fixed it so that at least 3 axis tick mark labels will always appear no matter how small the graphs get (see below)

![image](https://user-images.githubusercontent.com/41062454/106555068-100f0500-64da-11eb-935e-3d9ae36c6eec.png)

In doing so, I also added a pytplot global option that can accept inputs to alter the axis tick density, which you can do with the following snippet:

`pytplot.tplot_options("axis_tick_num", [(0,1000000), (3, 0.5)])`

This is entirely based on pyqtgraph's "textFillLimits" option for their axes: https://pyqtgraph.readthedocs.io/en/latest/graphicsItems/axisitem.html

Basically what the above is saying is: If you have 1, 2, or 3 tick marks, ensure that they do not fill more than 100000000% of the axis height (this essentially never happens, thus ensuring that at least 3 tick marks are displayed).  Once you get beyond 3 tick marks, ensure that any more tick marks it tries to add don't go over 50% of the axis height.  

In order to customize the plot labels exactly, it'll take a bit of toying around with to get the right numbers.  However, for now the default is to always display at least 3 tick marks.
